### ISSUE TITLE
Data Analyses

### ISSUE THREAD

**Issue by SlowMo24** [geopython/geopython-workshop#110]
The atmospheric data plotting (graph not map) seemed unrelated/unconnected to the rest of that. Maybe it is legacy content?

**Comment by justb4** [comment]
which chapter, 6?

**Comment by SlowMo24** [comment]
yes, if I remember correctly it the chapter is about spacial data but the first atmospheric task is about a time series with no geographic component
